database/things/click.pldb

```diff
@@ -1,8 +1,8 @@
 title Click
 appeared 1999
 type pl
-originCommunity Massachusetts Institute of Technology
 country United States
+originCommunity Massachusetts Institute of Technology
 
 features
  hasLineComments true
```

Comment:
Is there no web link for this language? If not, how do know the language features?

Reply:
There's a [Git repo](https://github.com/kohler/click). I'm guessing this file is about [this](https://github.com/kohler/click/wiki/Language). Should we add this link?